README.markdown

```diff
@@ -125,7 +119,14 @@ export NVM_DIR="$HOME/.nvm" && (
 ) && . "$NVM_DIR/nvm.sh"
 ```
 
-Add these lines to your `~/.bashrc`, `~/.profile`, or `~/.zshrc` file to have it automatically sourced upon login:
+Or if you have `git` installed (requires git v1.7+):
+
+1. clone this repo in the root of your user profile
+  - `cd ~/` from anywhere then `git clone https://github.com/creationix/nvm.git`
+1. check out the latest version
```

Comment:
let's provide the command here `git checkout v0.33.1` (the version number is updated in this file on every release)